### DISCUSSION TITLE
How to keep the completion table updating while typing without it to be closed for command to work as the same as option completion?

### DISCUSSION THREAD

**Discussion by IceAsteroid** [marlonrichert/zsh-autocomplete#762] [Q&A]
Hi, everyone.

I found no way to make zsh-autocomplete to automatically show and constantly update the completion table while typing command at the beginning of the prompt, I have to hit `tab` each time for completion table to show up.

While zsh-autocomplet works as expected for me for option completion, but it doesn't work the same for command completion.

Is it possible to make it work as for option completion?

Here's the config:
```
# Created by newuser for 5.9

# Bindkey to Emacs scheme, set this before not after zsh-autocomplete.
bindkey -e

# Download Znap, if it's not there yet.
znapDir=~/.znapRepos
znapFile=${znapDir}/znap/znap.zsh
[[ -r ${znapFile} ]] ||
    git clone --depth 1 -- \
        https://github.com/marlonrichert/zsh-snap.git ${znapDir}/znap
# Source znap into zsh
source ~/.znapRepos/znap/znap.zsh

# # `znap source` starts plugins.
znap prompt sindresorhus/pure
znap source marlonrichert/zsh-autocomplete
# zsh-autosuggestions must source after zsh-autocomplete
znap source zsh-users/zsh-autosuggestions

# # Configure marlonrichert/zsh-autocomplete
zstyle ':autocomplete:*' min-input 1
# zstyle ':autocomplete:*' default-context history-incremental-search-backward
bindkey              '^I'         menu-complete
bindkey "$terminfo[kcbt]" reverse-menu-complete
```

Here's the demo:

https://github.com/user-attachments/assets/37ff1a00-8e36-4312-9193-b080f2fa5891

**Comment by marlonrichert** [comment]
I think your command completion is probably timing out. You can test how long command completion takes by pressing down arrow and waiting for the menu to open.

If that is indeed the problem, you can try [increasing the timeout value](https://github.com/marlonrichert/zsh-autocomplete?tab=readme-ov-file#wait-longer-before-timing-out-autocompletion).
